src/sentry/api/endpoints/project_details.py

```diff
@@ -501,6 +501,9 @@ def get(self, request: Request, project) -> Response:
         if "hasAlertIntegration" in expand:
             data["hasAlertIntegrationInstalled"] = has_alert_integration(project)
 
+        if features.has("organizations:am2-billing", project.organization, actor=request.user):
```

Comment:
Not really because the problem that I am trying to solve is once a user was in LA/EA and now they are on the developer am2 plan, I want to hide `dynamicSampling` that was in EA/LA

Reply:
I think adding a developer flag makes sense in this case probably